apps/frontend-manage/src/pages/index.tsx

```diff
@@ -64,6 +71,26 @@ function Index() {
   const [isQuestionCreationModalOpen, setIsQuestionCreationModalOpen] =
     useState(false)
 
+  const dropdownItems = [
+    { value: 'CREATED', label: 'Datum' },
+    { value: 'TITLE', label: 'Titel' },
+  ]
+
+  const [sortBy, setSortBy] = useState('')
+
+  const sortIcon = useMemo(() => {
```

Comment:
Do we need 3 different icons? The list will be order in one way of the other initially anyways, so we could directly display this icon as an initial value. And then we can solve it inline at the TextField component with something like `sortBy?.asc ? faSort : faSortAsc` or similar?

Reply:
![image](https://user-images.githubusercontent.com/395846/215288746-3def7618-afb9-4314-947a-1cc9b3559873.png)

On the table on the evaluation, we have the same behavior to show whether sorting is active on a column. While it is necessary there to distinguish which columns are sorted (and not here with basically "one column"), I would prefer using the same behavior and the same icons (see https://github.com/uzh-bf/design-system/blob/25e2d37166273a7d89af9ffbb5b72da230fd6fba/src/Table.tsx#L147).

Reply:
makes sense :)